ValveResourceFormat/Resource/ResourceTypes/Material.cs

```diff
@@ -93,6 +96,13 @@ public override void Read(BinaryReader reader, Resource resource)
                 arguments.Add(name, value != 0);
             }
 
+            var specialDeps = (SpecialDependencies)Resource.EditInfo.Structs[ResourceEditInfo.REDIStruct.SpecialDependencies];
```

Comment:
do we really have to use editinfo for this? Is there not some flag in the material DATA that we can use instead (they are automatically loaded into the shader)

Reply:
I don't see any. At least from that VRF shows.